src/components/scenes/CryptoExchangeScene.tsx

```diff
@@ -408,7 +382,7 @@ export class CryptoExchangeComponent extends React.Component<Props, State> {
             secondaryCurrencyInfo={fromSecondaryInfo}
             fiatPerCrypto={this.props.fromFiatToCrypto}
             overridePrimaryExchangeAmount={this.state.fromExchangeAmount}
-            forceUpdateGuiCounter={this.state.forceUpdateGuiCounter}
+            forceUpdateGuiCounter={0}
```

Comment:
This prop is now unused. You can delete it from `CryptoExchangeFlipInputWrapper` and even `ExchangedFlipInput` itself.